<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JustMusic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>961</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="musicsButton">
         <property name="text">
          <string>Musics</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="albunsButton">
         <property name="text">
          <string>Albuns</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="artistsButton">
         <property name="text">
          <string>Artists</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPlaylistButton">
         <property name="text">
          <string>+ Playlist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="settingsButton">
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="musicsPage">
        <widget class="QTableWidget" name="musicsView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>961</width>
           <height>491</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="albunsPage">
        <widget class="QTableWidget" name="albunsView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>961</width>
           <height>491</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="artistsPage">
        <widget class="QTableWidget" name="artistsView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>961</width>
           <height>491</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="prevButton">
         <property name="text">
          <string>Prev</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="startLabel">
         <property name="text">
          <string>0:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="musicProgressBar">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="endLabel">
         <property name="text">
          <string>1:59</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shuffleButton">
         <property name="text">
          <string>Shuffle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="repeatButton">
         <property name="text">
          <string>Repeat</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>